src/wristpy/processing/metrics.py

```diff
@@ -0,0 +1,27 @@
+"""Calculate base metrics, anglez and enmo."""
+
+import numpy as np
+
+from wristpy.core.models import Measurement
+
+
+def euclidean_norm(acceleration: Measurement)-> Measurement:
+    """Compute ENMO, the Euclidean Norm Minus One (1 standard gravity unit).
+
+    Args:
+        acceleration: the three dimensional accelerometer data. A Measurement object,
+        it will have two attributes. 1) measurements, containing the three dimensional 
+        accelerometer data in an np.array and 2) time, a pl.Series containing 
+        datetime.datetime objects.
+
+    Returns:
+        A Measurement object containing the calculated ENMO values and the 
+        associated time stamps taken from the input.
+    """
+    #compute euclidean norm - 1
+    enmo = np.linalg.norm(acceleration.measurements, axis = 1) - 1
+
+    #GGIR replaces negative values with 0
```

Comment:
Try to avoid in-line comments; no need to say what the code does when the code already speaks for itself :). In-line comments should be used for the rare exception that there's no way to write the code clearly.

That being said, decisions like removing negative numbers do warrant some explanation. This would generally go in the docstring though i.e. between the top-level summary sentence and the Args block e.g. something like:

```python
def euclidean_norm(acceleration: Measurement)-> Measurement:
    """Compute ENMO, the Euclidean Norm Minus One (1 standard gravity unit).

    Negative ENMO values are set to zero because [add reason here; "GGIR does it" is not a reason ;) ] 

    Args:
        acceleration: the three dimensional accelerometer data. A Measurement object,
        it will have two attributes. 1) measurements, containing the three dimensional 
        accelerometer data in an np.array and 2) time, a pl.Series containing 
        datetime.datetime objects.
    ...
    """
 ```